### PR TITLE
Updated Image and Input component

### PR DESCRIPTION
**Pull request by Nagashree01**
added Image
- responsive image
- round image

Image

added Input
- textbox
- with validation and errors style
Input Text

### CODE REVIEW

**Review by Kothalkarpriya — APPROVED** [review]
Remove the unnecessary space from code and use formatting it will look better

**Comment by Nagashree01** [comment]
Yes I'll do this

File: components/input.css [hunk 1]
```diff
@@ -0,0 +1,40 @@
+@import url('/stylesheets/base.css');
+
+.input-area {
+    display: flex;
+    flex-direction: column;
+    gap: 1rem;
+    align-items: steach;
```
Comment (ozairkhan2810) [inline components/input.css:7]: stretch maybe

**Review by ozairkhan2810 — APPROVED** [review]
looks good, just one typo

File: components/input.css (hunk 1 above)
Comment (Nagashree01) [reply components/input.css]: oh yes Thank you

**Pull request merged**